### ISSUE TITLE
Version of FFmpeg to use?

### ISSUE THREAD

**Issue by gaviriar** [jkuri/opencv-ffmpeg-rtmp-stream#5]
Hi There,

I have a quick question. What version of FFmpeg was used to build this repo?

I am currently on ffmpeg 2.8 installed via `apt` and I cannot build it. The missing interfaces are present in ffmpeg 3.4 is that the version that should be used?

**Comment by jkuri** [comment]
yes, at least version `3.4` is required, preferably `4.0` if possible.

**Comment by gaviriar** [comment]
Ok, cool thanks for the clarification. I will test it out with version 3.4 and let you know. Might be worth leaving a note of the version dependency somewhere in the README.md or maybe setting the version in the CMakeLists.txt when looking for the module?

In case anyone else is facing this problem on Ubuntu 16.04 and needs to upgrade to version 3.4 here is how I did it:

```
sudo add-apt-repository ppa:jonathonf/ffmpeg-3
sudo apt update && sudo apt upgrade
```

**Comment by jkuri** [comment]
no problem. other option, if running some old version of distro, would be to run inside docker container using `ubuntu:bionic` image and apply `--device /dev/video0` when running container.

**Comment by gaviriar** [comment]
Ok, thanks I guess that can be another option. Although I would prefer not having to have to run a docker container to be honest.  But will definitely keep it in mind. Thanks!
